packages/x-charts-pro/src/internals/plugins/useChartProZoom/initializeZoomInteractionConfig.ts

```diff
@@ -18,21 +18,23 @@ export const initializeZoomInteractionConfig = (
       pinch: { type: 'pinch', requiredKeys: [], mouse: {}, touch: {} },
     };
   } else {
-    defaultizedConfig.zoom = initializeFor<'zoom'>(zoomInteractionConfig.zoom);
+    defaultizedConfig.zoom = initializeFor('zoom', zoomInteractionConfig.zoom);
   }
 
   if (!zoomInteractionConfig?.pan) {
     defaultizedConfig.pan = {
       drag: { type: 'drag', requiredKeys: [], mouse: {}, touch: {} },
+      wheel: { type: 'wheel', requiredKeys: [], mouse: {}, touch: {}, allowedDirection: 'x' },
```

Comment:
@alexfauquette I've enabled the behaviour by default. Try it out and let me know what you think.

Reply:
An option would be to detect the general direction the scroll is going, and if it is horizontal or vertical we pick that direction, if it is mostly diagonal, we discard it, though Im not sure the behaviour would be nice

Reply:
So, pan would be a different direction on windows/linux vs macos. This is expected though, as it follows the same behaviour as the "scroll", so I don't think it warrants a warning.

I've changed the pan `wheel` to only be enabled by default when `only` pan-x is enabled. If `pan-y` is also enabled we disable it to prevent weird behaviour where panning would only work in one direction.

I think this works nice for our current charts here, but let me know if you have any concern.

@prakhargupta1 ^